### ISSUE TITLE
Exercise 1 fails to verify

### ISSUE THREAD

**Issue by amahfouz** [azat-co/expressworks#43]
0. Running node v0.10.30 on Windows 7
1. Installed express.
2. Installed expressworks.
3. Wrote the simple program

```javascript
var express = require('express')
var app = express()

app.get('/', function(req, res) {
  res.write('Hello World!')
  res.end()
})
app.listen(process.argv[2])
```
And when I run:
`expressworks verify myprog.js`
   
it fails with the following message

```
events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: connect ECONNREFUSED
    at errnoException (net.js:904:11)
    at Object.afterConnect [as oncomplete] (net.js:895:19)
```

*. Ran the same program with a hardcoded 3000 port and invoking localhost:3000 worked from the browser and showed the "Hello World!".

**Comment by lagreenTW** [comment]
@ardooe 

me, too :)

$ ls
node_modules  program.js

**Comment by brookinc** [comment]
I had the same issue as @amahfouz (I opened the above nodeschool ticket before I found this one and realized the source of the issue). Increasing the timeout resolved it for me as well, although since I'm running a pretty slow setup (under vagrant on an old MacBook), I had to set it to 4500 -- anything less timed out. :(

I also noticed that the recent refactoring from setup.js to exercise.js changed the timeout value from 1500 back to 500 -- although even 1500 would have been too short in my case.
